src/main/java/org/stellar/sdk/StrKey.java

```diff
@@ -428,4 +493,33 @@ private static boolean isInAlphabet(final byte[] arrayOctet) {
     }
     return true;
   }
+
```

Comment:
minor, can you pull this out to it's own file `VersionByte.java`

Reply:
I think placing it here would also be a good choice, as there are no other parts relying on StrKey. Moreover, it is not public, so in the future, if necessary, we can still modify it without introducing breaking changes.